libs/sdk-ui-dashboard/package.json

```diff
@@ -81,6 +81,7 @@
         "immer": "^9.0.3",
         "json-stable-stringify": "^1.0.1",
         "lodash": "^4.17.19",
+        "moment": "^2.24.0",
```

Comment:
I thought we wanted to use `date-fns` instead of moment.js. If possible, try to use that.

Reply:
Great point, I missed the library. I have changed the parsing to proposed one.